specification/columns/availabilityzone.md

```diff
@@ -2,7 +2,7 @@
 
 An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
-The AvailabilityZone column MUST be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
+The AvailabilityZone column is OPTIONAL, but MUST be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Comment:
I believe this should be RECOMMENDED. The intent was to not require AvailabilityZone for the 1.0 timeframe. 
```suggestion
The AvailabilityZone column is RECOMMENDED to be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Reply:
My view is that we keep the MUST, SHOULD wording. Allow NULL and add some information around, if you dont have this column or data related to this column, either add the column with NULL or exclude the column.

Reply:
Alternatively we could use MAY instead of MUST here. That's duplicative of OPTIONAL, but conveys both points. RECOMMENDED or SHOULD is probably simpler.
```suggestion
The AvailabilityZone column is OPTIONAL, but MAY be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Reply:
Mostly agree with this, except the implementation of the idea that I think has the most clarity from an adoption perspective and the least friction would be to use create a default value such that if the value is not flipped certain assumptions are made about the conditions which can be changed by specifying just what the user wants to change.